### ISSUE TITLE
No bundle url present, xcode

### ISSUE THREAD

**Issue by jSkrills** [unimodules/react-native-unimodules#33]
Hey all :)  I am basically just trying to use expo's permissions with a bare project which led me to this [documentation](https://github.com/expo/expo/tree/master/packages/expo-permissions) which pointed me to the [unimodule docs](https://github.com/unimodules/react-native-unimodules) which I followed very close despite doing a few things that aren't documented. Like in my podfile I had to explicitly comment out EXPermissions, EXFileSystem, EXConstants, and EXAppLoaderProvider. Also I tried to directly point at my node_modules like this: `use_unimodules!(modules_paths: ['../node_modules'])` instead of the out of the box way.

After doing these things out of the box I still had to make some undocumented changes in EXConstantsBinding to declare UMInternalModule instead of EXInternalModule and change `EX_REGISTER_APP_LOADER(react-native-experience);` to `UM_REGISTER_APP_LOADER(react-native-experience);` in my EXHeadlessAppLoader.m file.

After I was finally able to get xcode to build successfully I ran into the 'No bundle URL present Make sure you're running a packager server or have included a .jsbundle file in your application bundle.' error and haven't been able to get around it through trial and error. 

In these [Advanced Expokit topics](https://docs.expo.io/versions/v32.0.0/expokit/advanced-expokit-topics/) I have verified that where my console tells me the app is running matches these locations in xcode: EXBuildConstants.plist (iOS) or ExponentBuildConstants. All I am doing to start things up is running expo start, which I believe is right. 

Basically I am a little lost out here trying to use unimodules and I don't know where I made a wrong turn. Help of any kind or guiding me towards more resources/reading would be very appreciated. Don't hesitate to ask me for more info/resources that I failed to provide around the issue. Thanks again!
